### PR TITLE
Switch to gRPC at Knative backend

### PR DESCRIPTION
**Pull request by boramalper**
Signed-off-by: Bora M. Alper <bora@boramalper.org>

### CODE REVIEW

**Comment by boramalper** [comment]
**gRPC:**
```
$ ./bin/word_count metamorphosis.txt
INFO[0000] Starting job0 (1/1)                          
Map 14 / 14 [=======================================================] 100.00% 0s
Reduce 1 / 1 [======================================================] 100.00% 0s
INFO[0000] Job 0 - Total Bytes Read:	838 kB             
INFO[0000] Job 0 - Total Bytes Written:	29 kB           
Job Execution Time: 138.027528ms
```

**HTTP:**
```
$ ./bin/word_count metamorphosis.txt
INFO[0000] Starting job0 (1/1)                          
Map 14 / 14 [=======================================================] 100.00% 0s
Reduce 1 / 1 [======================================================] 100.00% 0s
INFO[0000] Job 0 - Total Bytes Read:	838 kB             
INFO[0000] Job 0 - Total Bytes Written:	29 kB           
Job Execution Time: 93.899599ms
```

FYI

File: corral.proto [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+service Corral {
+    rpc Invoke(CorralRequest) returns (CorralRespone) {}
+}
+
+message CorralRequest {
+    bytes payload = 1;
+}
+
+message CorralRespone {
```
Comment (ustiugov) [inline corral.proto]:
```suggestion
message CorralResponse {
```

File: corral.proto [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+*/
+
+syntax = "proto3";
+
+option go_package = "github.com/bcongdon/corral";
+
+package corral;
+
+service Corral {
+    rpc Invoke(CorralRequest) returns (CorralRespone) {}
```
Comment (ustiugov) [inline corral.proto]:
```suggestion
    rpc Invoke(CorralRequest) returns (CorralResponse) {}
```

**Review by ustiugov — CHANGES_REQUESTED** [review]
a typo, otherwise lgtm

**Review by ustiugov — APPROVED** [review]

**Pull request merged**